fonts/leland/metadata.json

```diff
@@ -22,23 +22,27 @@
         "stemThickness": 0.11,
         "subBracketThickness": 0.11,
         "textEnclosureThickness": 0.11,
+        "textFontFamily": [
+            "Edwin",
+            "serif"
+        ],
```

Comment:
this too is part of #8853, but rnamed leland_metadata.json (on sync with the leland repo)